Templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+# Introduction
```

Comment:
Make it clear that this file needs changing, and should only be used as reference for the projects CONTRIBUTING.md file